framework/platform/platform.cpp

```diff
@@ -151,6 +151,11 @@ ExitCode Platform::main_loop_frame()
 		}
 
 		window->process_events();
+
+		if (window->should_close() || close_requested)
```

Comment:
Shouldn't the `active_app` be finished on `window->should_close()` as well?

Reply:
All I did was move the `window->should_close()` check _as-is_ from `main_loop()` to `main_loop_frame()`.  This check is typically used at the end of a batch run to shut things down.  Given that iOS never calls `main_loop()` and instead uses its own `renderLoop()` containing `main_loop_frame()`, this logic was missing on that platform.  By placing the check inside `main_loop_frame()`, all platforms now behave the same way on batch mode shutdown.

I am not sure I understand the condition you are asking about in your question.  Would you elaborate?  Thanks.